consensus/polybft/fsm_test.go

```diff
@@ -504,29 +501,11 @@ func TestFSM_VerifyStateTransactions_EndOfEpochWrongCommitEpochTx(t *testing.T)
 	testValidators := validator.NewTestValidators(t, 6)
 	allAccounts := testValidators.GetPublicIdentities()
 
-	fsm := &fsm{isEndOfEpoch: true, commitEpochInput: createTestCommitEpochInput(t, 0, allAccounts, 10), maxRewardToDistribute: maxRewardToDistribute, parent: &types.Header{}}
+	fsm := &fsm{isEndOfEpoch: true, commitEpochInput: createTestCommitEpochInput(t, 0, allAccounts, 10), parent: &types.Header{}}
 	commitEpochInput, err := createTestCommitEpochInput(t, 1, allAccounts, 5).EncodeAbi()
 	require.NoError(t, err)
 
-	commitEpochTx := createStateTransactionWithData(1, contracts.HydraChainContract, commitEpochInput, maxRewardToDistribute)
-	assert.ErrorContains(t, fsm.VerifyStateTransactions([]*types.Transaction{commitEpochTx}), "invalid commit epoch transaction")
-}
-
-func TestFSM_VerifyStateTransactions_EndOfEpochWrongCommitEpochTxValue(t *testing.T) {
```

Comment:
Why is this test removed?

Reply:
Because the test is related to the distribution value, so, as we don't send any funds anymore, the test becomes redundant.